content/cli.md

```diff
@@ -83,15 +84,38 @@ Or, if you want to build it yourself, the Golang source can be found [Here](http
 Once you have installed the cf-cli, we need to log in. In this sandbox environment, we need to log in though our Single Sign On portal. To start the sign on, run:
 
 ```bash
-cf login -a < TODO  url> --sso 
+cf login -a < TODO  url> -u <Email used in developer portal> 
 ```
-TODO: Fact Check
-This will give you a URL to browse to and log in. After you log in, you will get a single use token to copy and paste back into your terminal.
 
-TODO: Maybe have a space that the samples live in as well as the standard dev/test/prod?
-With that terminal, you should get the option of which Organization and Space you want to use. For now, select "samples". 
+This will prompt you for your password. This is the random password delivered along with your welcome email. (If you have since changed your password on Stratos, use what you changed it to)
```

Comment:
```suggestion
This will prompt you for your password. This is the random password delivered along with your welcome email. Remember to use your updated password in case you changed it in Stratos. 
```